Web_Development/FrontEnd/Semantic_HTML/semantic_html.md

```diff
@@ -0,0 +1,154 @@
+</main>
+```
+
+## HTML ```<footer>``` Element
+The ```<footer>``` HTML element represents the footer for its nearest sectioning content.It typically contains information about copyright details,author of the section or links to related documents and references.
+```html
+<footer>
+  <p>Author: abc</p>
+  <p><a href="mailto:abc@gmail.com">abc@gmail.com</a></p>
+</footer>
+```
+
+## HTML ```<nav>``` Element
+The ```<nav>``` element defines a set of navigation links for a website.
+```html
+<nav>
+  <a href="/home/">Home</a> |
+  <a href="/about/">About</a> |
+  <a href="/tools/">Tools</a> |
+  <a href="/contact/">Contact</a>
+</nav>
+```
+## HTML ```<aside>``` Element
+The ```<aside>``` HTML element represents a portion of a document whose content is only indirectly related to the document's main content. Asides are frequently presented as sidebars or call-out boxes.
+
+Display some content other than the content it is placed in:
+
+```html
+<p>My family and I visited The Kids center this weekend. The weather was nice, and he center was amazing! I had a great summer together with my family!</p>
+
+<aside>
+<h4>Kids Center</h4>
+<p>Kid center is a fun place to visit with family and kids. It has lots of stuff that children enjoy.</p>
+</aside>
+```
+
+## HTML ```<figure>``` and ```<figcaption>``` Elements
+The ```<figure>``` tag specifies self-contained content, like illustrations, diagrams, photos, code listings, etc.
+The ```<figcaption>``` HTML element represents a caption or legend describing the rest of the contents of its parent ```<figure>``` element.
+
+```html
+<figure>
+    <img src="flower.jpg" alt="flower">
+    <figcaption>A red rose</figcaption>
+</figure>
+```
+## Below is a list of some of the semantic elements in HTML.
+
+| Tag  | Description |
+| ------------- | ------------- |
+| ```<article>```  | Used for independent, self-contained content |
+| ```<aside>```  |  Used for content aside from the page content  |
+| ```<details>```  |  Used for additional details that the user can view or hide |
+| ```<figcaption>```  | Used for caption for a <figure> element |
+| ```<figure>```  | Specifies self-contained content, like illustrations, diagrams, photos, code listings, etc. |
+| ```<footer>```  | Used for footer for a document or section |
+| ```<header>```  | Used for header for a document or section |
+| ```<main>```  | Used for the main content of a document |
+| ```<mark>```  | Used for marked/highlighted text |
+| ```<nav>```  | Used for navigation links |
+| ```<section>```  | Used for section in a document |
+| ```<summary>```  | Used for visible heading for a ```<details>``` element |
+| ```<time>```  | Used for adate/time |
+
+### References  
+    https://developer.mozilla.org/en-US/docs/Glossary/Semantics
+    https://www.w3schools.com/html/html5_semantic_elements.asp
+    photo: 
+    https://miro.medium.com/max/1000/1*zHJFnu7QF-PgUb8108aLcA.png

```

Comment:
This thing looks like code-block. Try to remove that.

Reply:
I have made the changes.